### PR TITLE
fix: alignment of the image seen on no search results on application page

### LINKED ISSUE #13949: [Bug]: No search results image is misaligned on applications page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When i search something on applications page and there are no results, the no results image shown is misaligned.
image

### Steps To Reproduce

1. Go to the applications page
2. Search for anything random making sure there are no results.
3. Look at the no results image, it is misaligned.

### Public Sample App

https://app.appsmith.com/applications

### Version

Cloud / Self Hosted

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

> Alignment of the image seen on no search results on the application page. 

Fixes #13949 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

> Checked the alignment of the image is as expected

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/no-results-image-align 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 56.7 **(0)** | 38.65 **(0)** | 36.18 **(0)** | 56.95 **(-0.01)**
 :red_circle: | app/client/src/sagas/InitSagas.ts | 48.66 **(-1.9)** | 33.85 **(-2.82)** | 58.33 **(0)** | 51.57 **(-2.38)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>

### CODE REVIEW

File: app/client/src/sagas/InitSagas.ts [hunk 1]
```diff
@@ -454,6 +454,32 @@ export function* initializeAppViewerSaga(
   //Delay page load actions till all actions are retrieved.
   yield put(fetchPublishedPageSuccess([executePageLoadActions()]));
 
+  if (toLoadPageId) {
```
Comment (ankitakinger) [inline app/client/src/sagas/InitSagas.ts:457]: Note: This got added because I took pull from master branch

**Review by albinAppsmith — DISMISSED** [review]

**Review by albinAppsmith — DISMISSED** [review]

**Review by albinAppsmith — APPROVED** [review]

**Pull request merged**